src/Sentry.Maui.CommunityToolkit.Mvvm/Sentry.Maui.CommunityToolkit.Mvvm.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>MAUI and Community Toolkit integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
+    <TargetFrameworks>net9.0;net8.0</TargetFrameworks>
+
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <UseMaui>true</UseMaui>
+
+    <!-- Skip warnings about including Microsoft.Maui.Controls... that should be done from the application -->
+    <SkipValidateMauiImplicitPackageReferences>true</SkipValidateMauiImplicitPackageReferences>
+    <RootNamespace>Sentry.Maui.CommunityToolkitMvvm</RootNamespace>
```

Comment:
question: Should the root namespace match the project/package name?

I.e. missing `.` before `Mvvm`.
Or is the difference intentional?